src/DynamoPackages/PackageLoader.cs

```diff
@@ -200,9 +202,45 @@ private void TryLoadPackageIntoLibrary(Package package)
             {
                 var dynamoVersion = VersionUtilities.PartialParse(DynamoModel.Version);
 
+                package.EnumerateAdditionalFiles();
+
+                // If the additional files contained an extension manifest, then request it be loaded.
+                var extensions = package.AdditionalFiles.Where(
+                    file => file.Model.Name.Contains("ExtensionDefinition.xml") || file.Model.Name.Contains("ViewExtensionDefinition.xml")).ToList();
+
+                AssemblyLoadContext pkgLoadContext = AssemblyLoadContext.Default;
+
+                bool isolateAnyPackage = DynamoModel.FeatureFlags?.CheckFeatureFlag("IsolateAnyPackage", false) ?? false;
+                bool isolateThisPackage = DynamoModel.FeatureFlags?.CheckFeatureFlag("IsolatePackage_" + package.Name, false) ?? false;
```

Comment:
Should check per version ?
probably would be helpful for IronPython which already has isolation. The current version should not be isolated by Dynamo probably...but all other could be...